src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,64 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    private Movie movie;
+    @OneToOne
+    private CinemaHall cinemaHall;
```

Comment:
Let's add this annotation for the variable showTime `@Column(name = "show_time")`